index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Bens Portfolio Page</title>
+    <link rel='stylesheet' type='text/css' href='css/style.css'>
+    <link rel='stylesheet' type='text/css' href='css/normalise.css'>
+</head>
+
+<body>
+    <div class='hero'>
+        <nav>
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#aboutMe">About Me</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#contactInfo">Contact Info</a></li>
+            </ul>
+        </nav>
+        <div class="title">
+            <h1>Ben Ashfield</h1>
+            <h2>Web Developer in the making</h2>
+        </div>
+    </div>
+
+    <div class="heading" id="aboutMe">
+        <h1>About Me</h1>
+        <h2>After working as a manager in the retail industry for 15 years I decided to change careers and become a web developer.
+            I am currently studying full stack development at the Mayden Academy over a 16 week course to learn about the skills and tools needed.</h2>
+        <h2>In my spare time I enjoy online gaming. I particularly enjoy the competitive environment, trying to outfox and outplay opposing players
+            as well as playing together with a team strategising and working towards accomplishing a goal.</h2>
+    </div>
+
+    <div class="portfolio" id="portfolio">
+        <h1 class="portfolioTitle">Portfolio</h1>
+        <main>
+            <div class="projectBox project1">
+                <div class="projectInfo">
+                    <h1>Mayden Logo Challenge</h1>
+                    <h2>First big exercise to make the Mayden Academy logo using just HTML and CSS</h2>
+                </div>
+            </div>
+            <div class="projectBox project2">
+                <div class="projectInfo">
+                    <h1>Pilot Shop</h1>
+                    <h2>First build of a responsive website</h2>
+                </div>
+            </div>
+            <div class="projectBox project3">
+                <div class="projectInfo"></div>
+            </div>
+            <div class="projectBox project4">
+                <div class="projectInfo"></div>
+            </div>
+            <div class="projectBox project5">
+                <div class="projectInfo"></div>
+            </div>
+            <div class="projectBox project6">
+                <div class="projectInfo"></div>
+            </div>
+        </main>
+    </div>
+
+    <div class="contactInfo" id="contactInfo">
+        <h1 class="contactTitle">Contact</h1>
+    </div>
+    <div class="contactSymbols">
+        <ul>
```

Comment:
contactSymbols <ul>'s margin is making the bottom of the contact links have a white space at the bottom. Could be a proble if you are attempting to have no white space at the bottom of screen

Reply:
height: 120px; seems to cover the bottom with blue. There might be a better fix.

Reply:
Changed height within CSS